src/EnergyPlus/Furnaces.cc

```diff
@@ -588,10 +588,6 @@ namespace Furnaces {
                                                                 thisFurnace.CoolingCoilSensDemand,
                                                                 thisFurnace.CoolingCoilLatentDemand,
                                                                 thisFurnace.OpMode,
-                                                                thisFurnace.WSHPRuntimeFrac,
-                                                                thisFurnace.MaxONOFFCyclesperHour,
-                                                                thisFurnace.HPTimeConstant,
-                                                                thisFurnace.FanDelayTime,
```

Comment:
Since latent degradation inputs are now specified by the coil (and are not properties of the parent), they don't need to be passed into each method called on the child object.